toonz/sources/tnztools/fullcolorbrushtool.cpp

```diff
@@ -173,6 +171,7 @@ void FullColorBrushTool::onCanvasSizeChanged() {
 //---------------------------------------------------------------------------------------------------
 
 void FullColorBrushTool::onColorStyleChanged() {
+  getApplication()->getCurrentTool()->notifyToolOptionsBoxChanged();
```

Comment:
This addition causes re-building the tool options bar every time you change the color of style. It seems to be one of the  causes of sluggish response.